hudi-common/src/main/java/org/apache/hudi/common/model/CompactionOperation.java

```diff
@@ -139,8 +139,8 @@ public String toString() {
     return "CompactionOperation{"
         + "baseInstantTime='" + baseInstantTime + '\''
         + ", dataFileCommitTime=" + dataFileCommitTime
-        + ", deltaFilePaths=" + deltaFilePaths
-        + ", dataFilePath=" + dataFilePath
+        + ", deltaFilePaths=" + deltaFileNames
+        + ", dataFilePath=" + dataFileName
```

Comment:
Also dataFilePaths and deltaFilePath -> dataFileNames and deltaFileName?

Reply:
Fixed.